Bug: timeconverter proc converts unix time to local timezone

### Bug description

This was caught in a flapping tests when ran from different timezones, the issue is related to unix time to local time. The converted time can vary by quite a bit and result in entirely different expected dates.

E.g. 

```
--- FAIL: TestTimestampConvertorKey_Process (0.00s)
    --- FAIL: TestTimestampConvertorKey_Process/from_unix_to_string (0.00s)
        timestampconvertor_test.go:305: process() diff =   record.Record{
              	Position:  s"<nil>",
              	Operation: s"Operation(0)",
              	Metadata:  nil,
            - 	Key:       record.StructuredData{"date": string("2021-05-18")},
            + 	Key:       record.StructuredData{"date": string("2021-05-19")},
              	Payload:   {},
              }
--- FAIL: TestTimestampConvertorPayload_Process (0.00s)
    --- FAIL: TestTimestampConvertorPayload_Process/from_unix_to_string (0.00s)
        timestampconvertor_test.go:607: process() diff =   record.Record{
              	... // 2 identical fields
              	Metadata: nil,
              	Key:      nil,
              	Payload: record.Change{
              		Before: nil,
            - 		After:  record.StructuredData{"date": string("2021-05-18")},
            + 		After:  record.StructuredData{"date": string("2021-05-19")},
              	},
              }
```

### Steps to reproduce

run make test in a timezone which will convert have the clock jump to the day before or after.

### Version

v0.5.2

--- a/pkg/processor/procbuiltin/timestampconvertor.go
+++ b/pkg/processor/procbuiltin/timestampconvertor.go
@@ -92,7 +92,7 @@ func timestampConvertor(
 			var tm time.Time
 			switch v := d[field].(type) {
 			case int64:
-				tm = time.Unix(0, v)
+				tm = time.Unix(0, v).UTC()
 			case string:
 				if format == "" {
 					return record.Record{}, cerrors.Errorf("%s: no format to parse the date", processorType)

